homework/content.js

```diff
@@ -1,37 +1,44 @@
 'use strict';
 
-(function() {
-  var _wrapper = document.querySelector('#note-content-wrapper');
+var ContentManager =(function() {
 
-  function start() {
-    window.addEventListener('note-open', function(event) {
-      var note = event.detail;
-      resetWrapper();
-      drawNote(note);
-    });
-  }
+  // constructor
+  var self = function () {
```

Comment:
It may not need to create a inner function as the constructor, while the `ContentManager` **is** the constructor. And to overwrite the keyword `self` is not so good.